doc/architecture/database_schemata.md

```diff
@@ -87,8 +87,62 @@ species }|--|| genus : ""
 genus }|--|| subfamily : ""
 subfamily }|--|| family : ""
 
+maps {
+  INT id PK
+  VARCHAR name "NOT NULL"
+  BOOLEAN is_variant "NOT NULL"
+  VARCHAR version_name
+  BOOLEAN is_inactive
+  DATE last_visit
+  INT honors
+  INT visits
+  INT harvested
+  DATE creation_date
+  DATE deletion_date
+  INT zoom_factor
+  BYTEA background
```

Comment:
Maybe I misunderstood something, but I always read the base layer as the layer were I would upload a picture of my (hypothetical) garden to be the basis for all further planning including the optional geodata to add it to the Nextcloud Geo Map. With alternatives for this layer allowed, what would stop me from also uploading a picture of a park in New Zealand? My map would no longer represent a single location. The base layer also affects all work on all the other layers since a flower planted on top of a hill in the garden could suddendly be sitting in a pond in the alternative base layer.

Reply:
> Maybe I misunderstood something, but I always read the base layer as the layer were I would upload a picture of my (hypothetical) garden to be the basis for all further planning

This is correct.

> including the optional geodata

Probably we have different understandings of "geodata" means. The base layer contains:

- orientation
- real length of lines etc. for georeferencing. This may or may not be an absolute position in the real world.

> to add it to the Nextcloud Geo Map.

The Nextcloud Map feature is mainly to indicate where users are. I don't think we should use the data from the base layer there. But you are right: it would be nice if also the maps could be shown there eventually, so it is a good idea to allow the user to add a coordinate for their map (but independent of the base layer functionality).

> With alternatives for this layer allowed, what would stop me from also uploading a picture of a park in New Zealand?

Nothing, if you want to copy some elements of the park in New Zealand, this is exactly how you would do it.

> My map would no longer represent a single location.

Exactly, that is why the coordinates of the map should be in a separate field and not retrieved from the base layer.

> The base layer also affects all work on all the other layers since a flower planted on top of a hill in the garden could suddendly be sitting in a pond in the alternative base layer.

When you upload alternative images, you basically only change the scale/orientation of the image, not of all the elements you already placed. This was poorly described in the use case, I fixed it in #251.

Reply:
> When you upload alternative images, you basically only change the scale/orientation of the image, not of all the elements you already placed.

That is exactly my point. If the elements placed stay static but the underlying picture changes, then there would be the possibility that elements are in a place on the alternative image were they logically would not make sense, e.g. coordinates of plant X stay the same but in the alternative image you can see a body of water in exactly that place. You would need to either place the elements in locations that make sense for both base images or create an alternative layer for each base layer. And that would be confusing for the users, especially in a shared map scenario, to keep track which (alternative) layer is supposed to be used with which base image.

Reply:
It is a complicated matter. I updated #251 again to clarify the different needs:

1. The base image might be of the same place, but is just a different picture/drawing/... In this case you basically only stretch and rotate the image until it fits into the same polygon as before (Alternative scenario 1.)
2. The user actually wants to change where all the elements in the database are (the image is right and not the currently placed elements). This is **not supported** as alternative. So there can be only one geometry for every element on every layer.

We should also talk with Luciano about this, especially the georeferencing. Base layer is actually one of our first use cases. But most likely not the part of changing the geometry.